samcli/commands/init/init_templates.py

```diff
@@ -215,7 +229,11 @@ def _get_manifest(self):
         try:
             response = requests.get(MANIFEST_URL, timeout=10)
             body = response.text
-        except (requests.Timeout, requests.ConnectionError):
+            # if the commit is not exist then MANIFEST_URL will be invalid, fall back to use manifest in latest commit
+            if response.status_code == 404:
+                raise ManifestNotFoundException()
```

Comment:
Is there a debug message for this case where this exception is handled? Otherwise can we add it here?